doc/environment.yml

```diff
@@ -1,15 +1,25 @@
 name: test_env_xgcm_docs
+channels:
+  - conda-forge
+  - defaults
+  - anaconda
```

Comment:
Why are you using anaconda and defaults here? In our [CI builds](https://github.com/xgcm/xgcm/blob/master/ci/environment.yml) `conda-forge` is sufficient.

We use [this numba channel](https://github.com/xgcm/xgcm/blob/33db3f210f67ff195b225e67c3326b8dd11ca998/ci/environment-upstream-dev.yml#L4) for upstream builds, but I hope this is not necessary here